__tests__/app.test.js

```diff
@@ -140,4 +116,60 @@ describe('GET /api/articles/:article_id', () => {
         .then((res) => {
             expect(res.body).toEqual({message: 'Bad Request'})
         })
-    });
\ No newline at end of file
+    });
+});
+describe('GET /api/articles/:article_id/comments', () => {
+    test('should return 200 OK status and an array of all comments for a specific article, sorted in descending order by created_at', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toBeSorted({ descending: true, key: 'created_at' })
+            expect(body.length).toBe(11)
+            expect(body).toMatchObject([
+                {
+                  comment_id: 5,
+                  body: 'I hate streaming noses',
+                  article_id: 1,
+                  author: 'icellusedkars',
+                  votes: 0,
+                  created_at: '2020-11-03T21:00:00.000Z'
+                },
+                {created_at: '2020-10-31T03:03:00.000Z'},
+                {created_at: '2020-07-21T00:20:00.000Z'},
+                {created_at: '2020-06-15T10:25:00.000Z'},
+                {created_at: '2020-05-15T20:19:00.000Z'},
+                {created_at: '2020-04-14T20:19:00.000Z'},
+                {created_at: '2020-04-11T21:02:00.000Z'},
+                {created_at: '2020-03-02T07:10:00.000Z'},
+                {created_at: '2020-03-01T01:13:00.000Z'},
+                {created_at: '2020-02-23T12:01:00.000Z'},
+                {created_at: '2020-01-01T03:08:00.000Z'}
+              ])
```

Comment:
Im not sure why this is being tested in this way. You should test that every comment has all of the expected properties.